docs/concepts/permission-model.md

```diff
@@ -0,0 +1,15 @@
+#### OLMv1 Permission Model
+
+Here we aim to describe the OLMv1 permission model in terms of RBAC defined for the ClusterExtension and RBAC contained in the operator bundle ClusterServiceVersion.
```

Comment:
Let's avoid talking specifically about registry+v1 bundle-specifics (like the CSV). We can talk more generally about:
1. The purpose of the service account specified in the ClusterExtension spec, which is to manage everything in (2) below.
2. The contents of the bundle, which may contain more service accounts and RBAC

```suggestion
Here we aim to describe the OLMv1 permission model. OLMv1 itself does not have permission to manage the installation and lifecycle of cluster extensions. Rather, it requires that each cluster extension specifies a service account that will be used to manage its bundle contents.

... 
```